library/windows/BUILD.gn

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build/packaging.gni")
+import("//library/engine.gni")
+import("//library/windows/glfw.gni")
+import("//library/windows/jsoncpp.gni")
+
+action("fetch_glfw") {
+  script = "//tools/dart_tools/bin/fetch_glfw.dart"
+  outputs = glfw_files
+  args = [
+    rebase_path(glfw_download_dir, root_build_dir),
+  ]
+}
+
+action("build_jsoncpp") {
+  script = "//tools/dart_tools/bin/build_jsoncpp.dart"
+  outputs = jsoncpp_files
+  args = [
+    rebase_path(jsoncpp_download_dir, root_build_dir),
+    "--debug",
+  ]
+}
+
+config("relative_dependency_headers") {
```

Comment:
The jsoncpp and glfw pieces should be separate. Also, the name here should be changed since it's also vending lib_dirs.